### DISCUSSION TITLE
MVP brain storm

### DISCUSSION THREAD

**Discussion by geeksesi** [CodeReviewPals/code-review-pals#8] [Ideas]
Hello there,
we deal with the MVP version and then start to lunch the product.

I'll talk about my idea and willing. feel free to tell about yours.

**Comment by geeksesi** [comment]
## who is the user
- somebody has a pull request ( in his repository on others ) and needs a senior to review the code and guide him to make it better. - we called `Junior`
- somebody has an open-source project with some open pull request and need somebody else to review the pull requests and help to merge them. - we called `Busy man`
- somebody who is willing to increase community and opensource activity  - we called `Reviewer`

## how users integrate
- Junior can make a direct request.
- Busy man can attach his repository then we will list his pull requests on the website
- Reviewer will watch the requests ( with filters ) and submit his review on Github/Gitlab/others 🤔

## automation things
- extract detail of the pull request, project, user for better filtering and experience.
- handle status of the pull request, open, closed, merged
- watch the repositories pull requests and list them on site.

**Reply by abbasudo** [reply]
isn't it better to simplify Junior and Busy man and Reviewer to actions not type of user? 
something like stack overflow. each user can make a `Request` and can `Respond` to others requests. each request can be labeled as `private repo` `open source` `review` `help` `code snippet` etc.

**Reply by geeksesi** [reply]
there not user type things.
we just called them. not typed them :))
